server/room.ts

```diff
@@ -29,212 +36,124 @@ export class Room {
       this.deserialize(roomData);
     }
 
-    setInterval(() => {
-      // console.log(roomId, this.video, this.roster, this.tsMap, this.nameMap);
+    this.roomInterval = setInterval(() => {
       io.of(roomId).emit('REC:tsMap', this.tsMap);
+      if (
```

Comment:
probably could be separate setIntervals as we may not want to tie emitting TS and advancing the queue to the same function

Reply:
done